resources/migrations/instance_analytics_views/content/v4/mysql-content.sql

```diff
@@ -113,6 +113,30 @@ select
             where most_recent
         ) as moderation on concat('card_', report_dashboard.id) = moderation.entity_qualified_id
 union
+select
+    document.id as entity_id,
+    concat('document_', document.id) as entity_qualified_id,
+    'document' as entity_type,
+    created_at,
+    updated_at,
+    creator_id,
+    name,
+    null as description,
+    collection_id as collection_id,
+    null as made_public_by_user,
+    null as is_embedding_enabled,
+    null as is_verified,
+    archived,
+    null as action_type,
+    null as action_model_id,
+    null as collection_is_official,
+    null as collection_is_personal,
+    null as question_viz_type,
+    null as question_database_id,
+    null as question_is_native,
+    null as event_timestamp
+    from document
+union
```

Comment:
the migrations should be at v5 not v4, and we shouldn't override mysql's migration here